tests/forms_tests/tests/test_formsets.py

```diff
@@ -1197,6 +1197,61 @@ def test_html_safe(self):
         self.assertTrue(hasattr(formset, '__html__'))
         self.assertEqual(str(formset), formset.__html__())
 
+    def test_can_delete_extra_formset_forms(self):
```

Comment:
I didn't know if this test is required - there are plenty of other tests for deleting `formsets`. However, part of me liked the symmetry of having this test next to the following one to show the different behavior.

Reply:
Let's keep it.